src/silly_sprites/opengl/renderer.cpp

```diff
@@ -0,0 +1,104 @@
+#include "renderer.hpp"
+#include "buffer_object.hpp"
+#include "shader_program.hpp"
+#include "vertex.hpp"
+
```

Comment:
do we have an empty line between includes?

Reply:
I don't know, doesn't really matter to me. I will delete the empty line there.